core/txpool/legacypool/legacypool_test.go

```diff
@@ -1195,35 +1198,42 @@ func TestPendingGlobalLimiting(t *testing.T) {
 func TestAllowedTxSize(t *testing.T) {
 	t.Parallel()
 
-	// Create a test account and fund it
-	pool, key := setupPool()
+	pool, _ := setupPool()
 	defer pool.Close()
 
+	// We use a deterministic ecdsa key to obtain signed transactions of pre-determined sizes.
+	key := &ecdsa.PrivateKey{
+		PublicKey: ecdsa.PublicKey{
+			Curve: elliptic.P256(),
+			X:     new(big.Int).SetBytes(common.Hex2Bytes("78e68ff485cc934e4096f24091931e91334d6841c6c2be93d1a930e59783902d")),
+			Y:     new(big.Int).SetBytes(common.Hex2Bytes("ca8db229299a4f937eeb0a36c451751cd59df978aceaaf190c28783c321cf267")),
+		},
+		D: new(big.Int).SetBytes(common.Hex2Bytes("e521e757356c1f197f704502dfa10333993e3705ca3d304d6a349461ed67ae71")),
+	}
```

Comment:
This statement can be replaced by `crypto.HexToECDSA("e521e757356c1f197f704502dfa10333993e3705ca3d304d6a349461ed67ae71")`

Reply:
Done, thanks for the tip 👍